index.js

```diff
@@ -344,6 +344,45 @@ class ScmBase {
     stats() {
         return {};
     }
+
+    /**
+     * Get an array of scm context (e.g. [github.com, mygitlab_gitlab])
+     * @method getScmContext
+     * @return {Promise}
+     */
+    getScmContext() {
```

Comment:
The function name is changed from `getScmContext` to `getScmContexts`. 
https://github.com/screwdriver-cd/screwdriver/issues/365#issuecomment-314705743